we use system ryml so the checkouts are missing, just pull it directly.
diff --git a/cmake/SearchForStuff.cmake b/cmake/SearchForStuff.cmake
index f7381e1..f1e6f11 100644
--- a/cmake/SearchForStuff.cmake
+++ b/cmake/SearchForStuff.cmake
@@ -258,7 +258,7 @@ endif()
 # for now.
 #add_subdirectory(3rdparty/rapidyaml/rapidyaml/ext/c4core/src/c4/ext/fast_float EXCLUDE_FROM_ALL)
 add_library(fast_float INTERFACE)
-target_include_directories(fast_float INTERFACE 3rdparty/rapidyaml/rapidyaml/ext/c4core/src/c4/ext/fast_float/include)
+target_include_directories(fast_float INTERFACE 3rdparty/fast_float/include)
 
 add_subdirectory(3rdparty/jpgd EXCLUDE_FROM_ALL)
 add_subdirectory(3rdparty/simpleini EXCLUDE_FROM_ALL)
